pkg/datamover/repository_server.go

```diff
@@ -52,7 +52,8 @@ func (rs *repositoryServer) Push(ctx context.Context, sourcePath, destinationPat
 	if err != nil {
 		return err
 	}
-	return kopiaLocationPush(ctx, destinationPath, rs.outputName, sourcePath, password)
+	_, err = kopiaLocationPush(ctx, destinationPath, rs.outputName, sourcePath, password)
```

Comment:
It looks like we are not consuming the value that is being returned from the function? How is that being used?

Reply:
It would be used while Testing Kopia Repository Server based changes